oral_history/scripts/process.py

```
@@ -8,3 +10,8 @@ def __init__(self, reports_dir):
 
     def run(self, file_group, file_name, item_ark):
         print(f"script('{file_group}', '{file_name}', '{item_ark}')")
+
+        # error detection goes here
+        # temporary example error, uncomment to view in console
+
+        # raise CommandError('file "%s" not found' % file_name)
```

@darrowcoucla Looks good to me, works as expected.  This should use an `f-string`, like `raise CommandError(f'file "{file_name}" not found')`, but since it's a commented-out placeholder example, no changes needed at this point.

Merging; thanks --Andy